src/main/java/com/facebook/openwifirrm/CustomJettyServerFactory.java

```diff
@@ -22,8 +22,15 @@
 import spark.utils.Assert;
 
 /**
- * Creates Jetty Server instances. Majority of the logic is taken
- * from JettyServerFactory with port logic added.
+ * Creates Jetty Server instances. Majority of the logic is taken from
+ * JettyServerFactory. The additional feature is that this class will actually
+ * set two connectors (original class doesn't set any connectors at all and
+ * leaves it up to the serivce start logic). Since we set two connectors here
+ * on the server, Spark uses the existing conectors instead of trying to spin
+ * up its own connectors. See
+ * main/java/spark/embeddedserver/jetty/EmbeddedJettyServer.java in spark. The
```

Comment:
You can use `@see EmbeddedJettyServer` tag for this (on its own line at the end of the javadoc comment)